ui/components/component-library/modal-content/modal-content.tsx

```diff
@@ -88,12 +89,11 @@ export const ModalContent = forwardRef(
             as="section"
             role="dialog"
             aria-modal="true"
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
             backgroundColor={BackgroundColor.backgroundDefault}
             borderRadius={BorderRadius.LG}
-            width={BLOCK_SIZES.FULL}
-            marginTop={12}
-            marginBottom={12}
-            padding={4}
+            width={BlockSize.Full}
             ref={modalDialogRef}
             {...modalDialogProps}
           >
```

Comment:
With these updates make sure to check existing Modal components in the codebase and update accordingly